include/sbat.h

```diff
@@ -8,6 +8,16 @@
 
```

Comment:
I wish Github had a way to comment on the commit messages, so I'll comment here.

shim me more or less follow what's described in https://www.kernel.org/doc/html/v4.17/process/submitting-patches.html#the-canonical-patch-format, please flollow that. Specially line wrapping at ~75 columns.

Also I think it's better if you use Peter Jones instead of @vathpela in the commit message.